packaging/conda/build.sh

```diff
@@ -1,3 +1,16 @@
 # !/bin/bash
-cmake --preset linux-gcc-release -DBUILD_JARS=OFF -DTESTING=OFF
-cmake --build --preset linux-gcc-release --parallel --target MODL --target MODL_Coclustering
+
+# On macOS, we have to build with the compiler outside conda. With teh conda's clang the following error occurs:
+# ld: unsupported tapi file type '!tapi-tbd' in YAML file '/Applications/Xcode_14.2.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/lib/libSystem.tbd' for architecture x86_64
```

Comment:
We'll not be sure that the fix works until we test its installation. I have doubts that compiling with the system's compiler will work because conda is made to work in its environment (but maybe I'm wrong)

Reply:
Yes indeed this is not the good way to do it. We have to test this package on an OS different than the one used in the build process e.g. macOS 12 Monterey.

Reply:
If this doesn't work we should give a shot to the fix mentioned here : https://computing.docs.ligo.org/conda/compiling/#sdk